<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="answer">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_0">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>520</y>
      <width>74</width>
      <height>74</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>74</width>
      <height>74</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>520</y>
      <width>74</width>
      <height>74</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>74</width>
      <height>74</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>74</width>
      <height>74</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>340</y>
      <width>74</width>
      <height>74</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>430</y>
      <width>74</width>
      <height>74</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>520</y>
      <width>74</width>
      <height>74</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>430</y>
      <width>74</width>
      <height>74</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_bt">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>74</width>
      <height>74</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_9">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>340</y>
      <width>74</width>
      <height>74</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QSlider" name="aline">
    <property name="geometry">
     <rect>
      <x>-120</x>
      <y>230</y>
      <width>871</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="bananaandapples_area">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>650</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
